src/http/BaseModule.h

```diff
@@ -113,8 +113,8 @@ class BaseModule {
     RequestBodyJSON& body() { return _body; }
 
     // Message loggers for the corresponding log levels
-    // TODO:UJ all of these defeat the purpose of using macros, warn and error are probably fine to keep
-    //         as they should rarely be called.
+    // TODO:DM-52998 all of these defeat the purpose of using macros, warn and error are probably
+    //         fine to keep as they should rarely be called.
```

Comment:
This is true. The functions are meant to provide a convenient abstraction for message reporting beyond what is done by the LSST logger. Some functions (like `warn()`) accumulate warnings in a special collection, which is reported back to callers of the REST service. They also include an identity of the REST service in the reporting. These functions are not called frequently.